doc/libraries.xml

```diff
@@ -35,6 +35,9 @@
     <category name="Correctness">
       <title>Correctness and testing</title>
     </category>
+    <category name="Error-handling">
+      <title>Error-handling and recovery</title>
```

Comment:
The hyphen shouldn't be there. "Error handling and recovery". Although I think that "Error handling and reporting" would be better. Not sure about the recovery part.